src/App.tsx

```diff
@@ -15,6 +16,7 @@ const App = () => (
                 </Route>
             </Routes>
             <footer>
+                <StationPrompt />
```

Comment:
This probably shouldn't be in the footer

Reply:
Changed it